incubator/solr/values.yaml

```diff
@@ -0,0 +1,97 @@
+---
+
+# Which port should solr listen on
+port: 8983
+
+# Number of solr instances to run
+replicaCount: 3
+
+# Settings for solr java memory
+javaMem: "-Xms2g -Xmx3g"
+
+# Set the limits and requests on solr pod resources
+resources: {}
+
+# Sets the termination Grace period for the solr pods
+# This can take a while for shards to elect new leaders
+terminationGracePeriodSeconds: 180
+
+# Solr image settings
+image:
+  repository: solr
+  tag: 7.6.0
+  pullPolicy: Always
```

Comment:
The `pullPolicy` should be `IfNotPresent` so that local caching can be used. Can you please update

Reply:
I have updated the `pullPolicy`.